### PR TITLE
3 get /api/categories

### PR DESCRIPTION
**Pull request by RossKeen**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          expect(Array.isArray(res.body)).toBe(true);
+          expect(res.body.length > 0).toBe(true);
+          res.body.forEach((categoryObject) => {
+            expect(categoryObject).toHaveProperty("slug");
+            expect(categoryObject).toHaveProperty("description");
+          });
+        });
+    });
+  });
+});
```
Comment (emilyb93) [inline __tests__/app.test.js:40]: would like to see a test for an invalid endpoint /api/categoriez or something similar

**Review by emilyb93 — CHANGES_REQUESTED** [review]
Needs another test

File: __tests__/app.test.js (hunk 1 above)
Comment (RossKeen) [reply __tests__/app.test.js:40]: ✅ - Implemented.

**Review by emilyb93 — APPROVED** [review]
Looks good!

**Pull request merged**